tests/test_legacy_invitations.py

```diff
@@ -47,7 +53,8 @@ def test_single_blind_conference(self, client, test_client, selenium, request_pa
         note.content['pdf'] = url
         test_client.post_note(note)
 
-        conference.create_paper_groups(authors=True, reviewers=True, area_chairs=True)
+        # conference.create_paper_groups(authors=True, reviewers=True, area_chairs=True)
+        conference.setup_final_deadline_stage()
```

Comment:
coud you check that after calling to the final deadline, the submissions become public?